core/drivers/scmi-msg/smt.c

```diff
@@ -54,9 +54,12 @@ struct smt_header {
 #define SMT_MSG_PROT_ID_MASK		GENMASK_32(17, 10)
 #define SMT_HDR_PROT_ID(_hdr)		(((_hdr) & SMT_MSG_PROT_ID_MASK) >> 10)
 
-static struct smt_header *channel_to_smt_hdr(struct scmi_msg_channel *chan)
+static struct smt_header *channel_to_smt_hdr(struct scmi_msg_channel *channel)
```

Comment:
I'm not sure renaming `chan` to `channel` helps much, but it sure make it harder to read this patch due to the churn.

Reply:
ok, i'll split the change into 2 commits.